helm/values.yaml

```diff
@@ -6,21 +6,21 @@ receive:
     type: ClusterIP
     port: 80
   service_tcp:
-    type: LoadBalancer
+    type: ClusterIP
```

Comment:
Since you're overriding these in the AWS version, can we make these guys NodePorts in the default? (Or override them in the helm values?)

Reply:
Done